package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "central-settlement",
     "description": "Central settlements hosted by a scheme to record and make settlements.",
-    "version": "5.2.0",
+    "version": "5.2.1",
```

Comment:
Needs to be 5.2.0. Please check - https://github.com/mojaloop/central-settlement/releases